Subject: Don't force select AES in wireless
From: ak@suse.de

x86 kernels use always the assembler optimized versions of AES and TWOFISH.
But the wireless stack would force enable the generic C aes anyways.
Remove that. The optimized versions provide the cipher as well.

---
 drivers/net/wireless/Kconfig |    1 -
 net/ieee80211/Kconfig        |    1 -
 net/mac80211/Kconfig         |    1 -
 3 files changed, 3 deletions(-)

Index: 2008-devel/drivers/net/wireless/Kconfig
===================================================================
--- 2008-devel.orig/drivers/net/wireless/Kconfig
+++ 2008-devel/drivers/net/wireless/Kconfig
@@ -445,7 +445,6 @@ config AIRO_CS
 	depends on PCMCIA && (BROKEN || !M32R) && WLAN_80211
 	select WIRELESS_EXT
 	select CRYPTO
-	select CRYPTO_AES
 	---help---
 	  This is the standard Linux driver to support Cisco/Aironet PCMCIA
 	  802.11 wireless cards.  This driver is the same as the Aironet
 	(aka TGi, WPA, WPA2, WPA-PSK, etc.) for use with CCMP enabled
Index: 2008-devel/net/mac80211/Kconfig
===================================================================
--- 2008-devel.orig/net/mac80211/Kconfig
+++ 2008-devel/net/mac80211/Kconfig
@@ -3,7 +3,6 @@ config MAC80211
 	select CRYPTO
 	select CRYPTO_ECB
 	select CRYPTO_ARC4
-	select CRYPTO_AES
 	select CRC32
 	select WIRELESS_EXT
 	select CFG80211
